main/src/ca/uwaterloo/flix/language/phase/Inliner.scala

```diff
@@ -246,6 +250,18 @@ object Inliner {
       val e2 = visitExp(exp2, subst0)
       LiftedAst.Expr.LetRec(varSym, index, defSym, e1, e2, tpe, purity, loc)
 
+    case OccurrenceAst.Expression.Stmt(exp1, exp2, tpe, purity, loc) =>
+      /// Case 1:
```

Comment:
Can this case actually happen or is it dead code?

Reply:
The optimizer loops so it will give itself Stmts.